docs/index.md

```
@@ -154,6 +154,11 @@
     </div>
 </div>
 
+<div class="get-started-section">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: div

(MD033, no-inline-html)